keps/sig-api-machinery/0006-apply.md

```diff
@@ -299,6 +301,77 @@ The conversion between the two and creating the diff was complex and would have
 
 - 12/2019 [#86083](https://github.com/kubernetes/kubernetes/pull/86083) implementing a poc for the described approach
 
+#### Fix Admission Chain for Subresources
+
+##### Current Behavior
+
+Currently, the admission chain on a subresource is not propagated for the parent
+resource.
+
+When a user makes a request to a subresource, then admission webhooks and
+validation are skipped on the parent, and we don't have the user information to
+set the field manager for the parent.
+
+For example, a validating webhook on Deployments to enforce a replica policy
+can be bypassed by any /scale subresource request because
+the webhook admission chain is not called on the subresource. We cannot track
+that the user who made the request to the /scale resource owns the replicas
+field.
+
+##### Proposed Change
+
+The webhooks will be called for the subresource and parent using the same
+admission context, such that the original user is represented in the admission
+attributes. Webhooks will opt-in to this behavior on subresources in the
+registration.
+
+The webhooks will be called in the following order, considering a request to
+`/foo/scale` as an example:
+1. mutating webhooks will be called on the included subresource (/foo/scale)
+2. mutating webhooks will be called on the parent (/foo)
+3. optionally, mutating webhooks will be called a second time in the same order
```

Comment:
Mention that this is already what we do for mutating webhooks AFAIK

Reply:
+1

Reply:
Done.